2996_reflection/reflection.md

```diff
@@ -3838,6 +3838,25 @@ namespace std::meta {
   template <reflection_range R1 = span<info const>, reflection_range R2 = span<info const>>
     consteval info reflect_invoke(info target, R1&& tmpl_args, R2&& args);
 
+  // [meta.reflection.define_class], class definition generation
+  struct data_member_options_t {
+    struct name_type {
+      template <typename T> requires constructible_from<u8string, T>
+        consteval name_type(T &&);
+
+      template <typename T> requires constructible_from<string, T>
+        consteval name_type(T &&);
+    };
+
+    optional<name_type> name;
+    optional<int> alignment;
+    optional<int> width;
+  };
+  consteval info data_member_spec(info type,
+                                  data_member_options_t options = {});
```

Comment:
Did we want to actually stick `type` into the options so that they're all uniform?

Reply:
I thought we came down on "no". For this question?
Anyway, I'm too tired.  We can change it another time.